### PR TITLE
Style Checking for Core

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Update .editorconfig from dotnet/corefx /.editorconfig
Fix code style to adhere to editorconfig rules

### CODE REVIEW

**Comment by pakrym** [comment]
@chidozieononiwu what was the reason for the build failure? Can you link the issue please?

**Comment by chidozieononiwu** [comment]
> @chidozieononiwu Dozie Ononiwu FTE what was the reason for the build failure? Can you link the issue please?

Its due to this issue #7490

File: sdk/core/Azure.Core/src/Diagnostics/HttpPipelineEventSource.cs [hunk 1]
```diff
@@ -43,7 +43,7 @@ internal sealed class HttpPipelineEventSource : EventSource
 
         private HttpPipelineEventSource() : base(EventSourceName) { }
 
-        internal static readonly HttpPipelineEventSource Singleton = new HttpPipelineEventSource();
+        internal static readonly HttpPipelineEventSource s_singleton = new HttpPipelineEventSource();
```
[the remark this answers is not in the document]
Comment (chidozieononiwu) [reply sdk/core/Azure.Core/src/Diagnostics/HttpPipelineEventSource.cs]: Fixed

**Review by pakrym — APPROVED** [review]

**Pull request merged**